src/vma/util/sys_vars.cpp

```diff
@@ -336,6 +336,41 @@ void mce_sys_var::read_env_variable_with_pid(char* mce_sys_name, size_t mce_sys_
 	}
 }
 
+bool mce_sys_var::check_cpuinfo_flag(const char* flag)
```

Comment:
@serglelv please explain why the flag is_hypervisor is a sufficient criteria for not calling the assembly code